ConsentViewController/Classes/SourcePointClient/MetaDataRequestResponse.swift

```diff
@@ -21,10 +21,15 @@ struct MetaDataResponse: Decodable, Equatable {
         let sampleRate: Float
     }
     struct USNat: Decodable, Equatable {
-        let _id: String
+        let vendorListId: String
```

Comment:
I'm gonna do the same

Reply:
@carmelo-iriti 👍  Keep in mind the API returns `_id`, I just thought it'd be more readable to change that internally to `vendorListId`.